plugins/mac-avcapture/av-capture.mm

```diff
@@ -2236,7 +2237,28 @@ static void add_manual_properties(obs_properties_t *props)
 		OBS_COMBO_FORMAT_STRING);
 	obs_property_list_add_string(dev_list, "", "");
 
-	for (AVCaptureDevice *dev in [AVCaptureDevice devices]) {
+	NSArray *devices = nil;
+#if (__MAC_OS_X_VERSION_MAX_ALLOWED >= MAC_OS_X_VERSION_10_15)
```

Comment:
Did we decide on that already? Lowest _supported_ macOS version should be 10.15 for x86_64-based systems, but we want to make the lowest supported _development macOS version_ (that is, the version used to compile OBS) be 11.0?

Reply:
I thought we decided on that (or rather generally on the current version and one lower) - though we didn't actively break anything since, so that decision could be reverted if we wanted.

Reply:
> Did we decide on that already? Lowest _supported_ macOS version should be 10.15 for x86_64-based systems, but we want to make the lowest supported _development macOS version_ (that is, the version used to compile OBS) be 11.0?

I also thought we had decided on this split (minimum 10.15 for runtime, minimum 11.0 for development/building). I'm fine either way and have no strong opinion on this particular bit of code.

Reply:
Alright, so here's my suggestion: Given that we don't have a PR merged that "officially" increases the lower limit to 10.15 across the code base (meaning raising that in CMake files, build scripts, workflow files), etc. I'd rather have this PR assume this to haven't happened yet and thus work on master branch "as-is".

I'm fine with removing this in a later pass (which we'd have to do anyway) when we remove this and other workarounds for older macOS versions.

Thoughts?

Reply:
Clarified off thread.